examples/prediction_error.py

```diff
@@ -0,0 +1,21 @@
+import matplotlib.pyplot as plt
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+from sklearn_evaluation import plot
+
+data = datasets.make_classification(200, 10, n_informative=5, class_sep=0.65)
```

Comment:
use `make_regression`, since `make_classification` will generate data for a classification problem